<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">/* VERTICAL SCROLLBAR */
QScrollBar:vertical {
	boder: none;
	background-color: rgb(178, 247, 200)
}
QScrollBar::handle:vertical:hover{
	background-color: rgb(101, 160, 105)
}

/* HORIZONTAL SCROLLBAR */
QScrollBar:horizontal {
	boder: none;
	background-color: rgb(178, 247, 200)
}
QScrollBar::handle:horizontal:hover{
	background-color: rgb(101, 160, 105)
}
</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1324</width>
         <height>824</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>800</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1" colspan="3">
              <widget class="QPushButton" name="Zakoncz">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Zakończ</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="Line" name="line_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="Line" name="line_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="Wykres1">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(170, 209, 185);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Wykres2">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(170, 209, 185);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="4">
              <widget class="Line" name="line_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QPushButton" name="QQ">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Wykres kwantyl-kwantyl</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Histogram">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Histogramy</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="SWtest">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Test Shapiro-Wilka (normalność rozkładu)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Ftest">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Test F-Scedecora (założenie o równości wariancji)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Ttest">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Test T-Studenta dla prób niezależnych</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="info">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Dane1">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(170, 209, 185);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Dane2">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(170, 209, 185);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="5">
              <widget class="QTextEdit" name="textEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="0" column="6">
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1362</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionDane1"/>
    <addaction name="actionDane2"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDane1">
   <property name="text">
    <string>Dane 1.</string>
   </property>
  </action>
  <action name="actionDane2">
   <property name="text">
    <string>Dane 2.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
